src/calculateRentalCost.js

```diff
@@ -1,10 +1,22 @@
+const carCosts = 40;
+
 /**
  * @param {number} days
  *
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return carCosts * days - 50;
```

Comment:
Instead of using a literal for the discount value, define a constant with a descriptive name to explain what this value represents. This applies to both the 7-day threshold and the $50 discount. For example, use `const LONG_TERM_RENTAL = 7;` and `const LONG_TERM_DISCOUNT = 50;`.